src/aap_eda/core/utils/credentials.py

```diff
@@ -342,3 +348,19 @@ def _validate_ssh_key(schema: dict, data: str, inputs: dict) -> list[str]:
         errors.append(str(e))
 
     return errors
+
+
+def _validate_gpg_public_key(key_data: str) -> list[str]:
+    errors = []
+
+    gpg = gnupg.GPG()
+    import_result = gpg.import_keys(key_data)
+
+    if import_result.returncode != 0:
+        errors.append("gpg: no valid OpenPGP data found.")
```

Comment:
@hsong-rh do we need the gpg: prefix. Also what happens if the key is an empty string? In the JIRA issue they are passing an empty string.

Reply:
The key is required fields and cannot be empty. Both UI and backend will exclude it.